dashboard/test/ui/features/platform/hamburger.feature

```diff
@@ -0,0 +1,430 @@
+@single_session
+Feature: Hamburger dropdown
```

Comment:
git is getting confused about me moving this file and indenting the scenarios.

I only added the last three that deal with the links for teacher, student, and signed out contexts.